### ISSUE TITLE
Dismantling requirements tables 29-36

### ISSUE THREAD

**Issue by tkoeppe** [cplusplus/draft#6033]
To convert the lib-intro requirements tables 29-36 into paragraphs, we don't currently have any section, and no good way to reference them.

Idea:

1. A new subsection "16.4.4.?: Basic type requirements", after [16.4.4.2, utility.arg.requirements] and before [16.4.4.3, swappable.requirements].
2. New nested subsections for each individual requirement, so that we can reference them individually (which is currently possible via the tables).

Thoughts?

**Comment by tkoeppe** [comment]
@jensmaurer, @jwakely feedback welcome!

**Comment by Dani-Hub** [comment]
Personally, I would strongly prefer to make each requirement individually referencable.

**Comment by tkoeppe** [comment]
@Dani-Hub Would it be sufficient to have a (very short) subsection for each requirement?

**Comment by tkoeppe** [comment]
Maybe we don't need a new grouping subsection, but instead we can just put each requirement straight into its own level-3 subsection, so we'd have new subsections 16.4.4.3 -- 16.4.4.9.

**Comment by Dani-Hub** [comment]
> @Dani-Hub Would it be sufficient to have a (very short) subsection for each requirement?

Yes, sure, we don't need extraordinary overhead. It is just the thing that we want to refer to them individually.

**Comment by tkoeppe** [comment]
Note to selves: the fpos.requirements table should be quite easy to dismantle, too.

**Comment by Dani-Hub** [comment]
> Note to selves: the fpos.requirements table should be quite easy to dismantle, too.

I'm eagerly awaiting that.

**Comment by tkoeppe** [comment]
@Dani-Hub PRs are of course always welcome :-)

**Comment by AlisdairM** [comment]
I will be looking into this in the New Year (2024) if no-one beats me to it.  It is annoying how often these requirements tables float into misleading subsections due to page breaking.
